src/utils/validation.ts

```diff
@@ -58,3 +58,9 @@ export const stringOrNumberResolverFactory = <FormData, CustomContext = {}>(
       : {},
   }
 }
+
+export function formatSchemaErrorMessage(errorString?: string): string | undefined {
+  if (!errorString) return undefined
+  const cleanedString = errorString.replace(/"/g, '')
```

Comment:
Why? How does it look without replace?

Reply:
`"spread" is dumb, change that shit` - there are some settings inside the API that allow me to change it but i couldnt figure it out